etna/transforms/timestamp/fourier.py

```diff
@@ -191,22 +269,19 @@ def _compute_features(self, timestamp: pd.Series) -> pd.DataFrame:
 
         return features
 
-    def _convert_sequential_timestamp_datetime_to_int(self, timestamp: pd.Series):
-        if self._reference_timestamp is None:
-            raise ValueError("The transform isn't fitted!")
-
+    def _convert_regular_timestamp_datetime_to_numeric(
+        self, timestamps: pd.Series, reference_timestamp: pd.Timestamp, freq: Optional[str]
```

Comment:
there is an issue with `timestamps` <-> `timestamp` naming above and below. Should we fix it?

Reply:
What is the issue? I don't see any.

Reply:
Here for example https://github.com/etna-team/etna/pull/223/files/cc88a3f73fc0fcf39f01b4872f780ae71a5c87a6#diff-c625b60860df44bc59ee41dad844da0f682e6313af23d33070144435f7593ed9R210

there are a lot of places where `timestamp` is a sequence not a scalar